src/foam/dao/PurgeRecordCmd.js

```diff
@@ -0,0 +1,12 @@
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'PurgeRecordCmd',
+  documentation: `
+    This model is passed to the dao cmd_ method to specify an individual
+    item to be purged from the cache.
+  `,
+
+  properties: [
+    'id'
```

Comment:
Add class: 'Object' so that it will be included in the java version so it can be marshalled over network.